src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1086,6 +1114,40 @@ export class MaximizeGroupAction extends Action2 {
 	}
 }
 
+export class UnmaximizeEditorGroupAction extends Action2 {
+
+	constructor() {
+		super({
+			id: EXIT_MAXIMIZE_EDITOR_GROUP,
```

Comment:
Please add this command id to `EDITOR_CORE_NAVIGATION_COMMANDS` which is an array of commands we ensure to always show in the toolbar, even when so many actions are present that the actions overflow into the "..." menu.

Reply:
Thats great! Was not aware this existed